src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -1,4 +1,49 @@
-const convertInclude = ({ href, url, type, ...rest }) => {
+import 'isomorphic-fetch';
+import nodeLogger from '#lib/logger.node';
+
+const logger = nodeLogger(__filename);
+
+// this defaults all VJ and IDT2 includes to the specified urls for now
+// should be removed once mozart routes have been created
+const includeUrls = {
+  vj: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/vj.html',
+  idt2: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/idt2.html',
+  idt1: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/idt1.html',
+};
+
+export const encodeHTML = str =>
+  String(str)
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&#39;');
+
+const fetchMarkup = async url => {
+  try {
+    const res = await fetch(url, { timeout: 2000 });
+    if (res.status !== 200) {
+      throw new Error('Failed to fetch');
+    } else {
+      const html = await res.text();
+      return encodeHTML(html);
```

Comment:
Would it make sense to pass the raw response straight into the component (as if it had fetched it) and then do our operations?